src/bundle/Resources/public/css/variables.css

```diff
@@ -25,13 +27,13 @@
     --ez-color-action-bar: #106d95; /* bad name, as it'll probably be used elsewhere */
     --ez-color-discovery-bar: #667076; /* bad name, as it'll probably be used elsewhere */
     --ez-color-button-focus: #fcb700;
-    --ez-color-button-focus-alt: #2b84b1; /* TODO: focus and focus-alt are inverted, fix the style guide */
+    --ez-color-button-focus-alt: #2b84b1;
 
     /* background colors */
-    --ez-color-bg: #fafafa;
-    --ez-color-bg-header: #e7e7e7; /* maybe okish, but would prefer a color name */
-    --ez-color-bg-content: #f3f3f3; /* bad name, as it'll probably be used elsewhere */
-    --ez-color-bg-section-main: #e3e3e3;  /* bad name, as it'll probably be used elsewhere */
-    --ez-color-bg-label: #dbdbdb; /* bad name, as it'll probably be used elsewhere */
-    --ez-color-bg-fieldtype: #bbb; /* bad name, as it'll probably be used elsewhere */
+    --ez-color-ground-base: #fafafa;
+    --ez-color-ground-base-header: #e7e7e7;
```

Comment:
forgot to remove some custom properties.
This is fixed in 13fd8ab9127a1e71ea1af05e64b7ca9e8f070406